ci/jenkins/test-vm.sh

```diff
@@ -294,8 +298,12 @@ function deliver_antrea_vm {
     export GOROOT=/usr/local/go
     export GOCACHE=${WORKSPACE}/../gocache
     export PATH=${GOROOT}/bin:$PATH
+    TEMP_ANTREA_TAR="antrea-image.tar"
 
     make docker-bin
+    docker save antrea/antrea-ubuntu -o $TEMP_ANTREA_TAR
```

Comment:
```suggestion
    docker save antrea/antrea-ubuntu:latest -o $TEMP_ANTREA_TAR
```
Otherwise it packages all tags of this image.

Reply:
updated